conf/receiver.cfg

```diff
@@ -25,17 +26,22 @@ check_crls: false
 
 [messaging]
 # If using AMS this is the project that SSM will connect to. Ignored for STOMP.
-ams_project:
+ams_project: accounting
 
 # Destination to which SSM will listen.
-destination: /queue/ssm2test
+destination: ssm2test
+
+# Only use direct token auth with AMS if you've been provided with a token to use.
+#token:
 
 # Accepted messages will be written to <path>/incoming
 # Rejected messages will be written to <path>/reject
 path: /var/spool/apel
 
 [logging]
 logfile: /var/log/apel/ssmreceive.log
+# Available logging levels:
+# DEBUG, INFO, WARN, ERROR, CRITICAL
```

Comment:
SSM uses at least debug and info (remember that verbosity PR?)  Would this not disrupt that?